packages/lib/onedrive-api.ts

```diff
@@ -265,11 +266,12 @@ export default class OneDriveApi {
 
 		for (let i = 0; i < 5; i++) {
 			options.headers['Authorization'] = `bearer ${this.token()}`;
+			options.headers['User-Agent'] = `ISV|Joplin|Joplin/${version}`;
 
-			const handleRequestRepeat = async (error: any) => {
+			const handleRequestRepeat = async (error: any, sleep_seconds: number = (i + 1) * 5) => {
```

Comment:
It feels weird to have variable in the default parameter (I'm surprised it even works!). Could you perhaps set it to null instead, then have in the function `sleepSeconds = sleepSeconds === null ? (i + 1) * 5 : sleepSeconds`

Also please use camelCase for the variable names, so should be "sleepSeconds"

Reply:
Have defaulted the parameter to null and implemented your suggestion using the null coalescing operator:
```javascript
sleepSeconds ??= (i + 1) * 5
```